### ISSUE TITLE
Question: Declaring permissions on Google Play

### ISSUE THREAD

**Issue by LegendEffects** [matinzd/react-native-health-connect#154]
Hello,

Apologies for sort of abusing the issues section as this doesn't directly relate to this package but there's no discussions tab on this repo and I'm running out of places to ask.

How have people been publishing their apps to Google Play? We've been trying along with contacting support for the last month to no avail, they've switched from the form to using the play console. Only problem is that to submit a request to get permission to use the required permissions you need an app release with those permissions, and we can't create an app release since we don't have permission to use those permissions even in the internal testing track...

Does anyone have any insights on what we're missing?

Thanks and apologies again for hijacking the issues section.

**Comment by matinzd** [comment]
Hey!

No worries. I have now enabled the discussions section of the repo. I did not realise that it's not enabled by default. Can you move it there?

**Comment by LegendEffects** [comment]
Yep, moved to discussions https://github.com/matinzd/react-native-health-connect/discussions/156 thanks
